Library/Homebrew/cask/artifact/moved.rb

```diff
@@ -77,7 +90,18 @@ def move(adopt: false, force: false, verbose: false, command: nil, **options)
           end
         end
 
-        if target.dirname.writable?
+        if target.directory?
+          if target.writable?
+            source.children.each { |child| FileUtils.move(child, target + child.basename) }
+          else
+            command.run!("/bin/cp", args: ["-pR", "#{source}/*", "#{source}/.*", "#{target}/"],
+                                    sudo: true)
+          end
+          unless Quarantine.copy_xattrs(source, target)
+            opoo "Unable to transfer extended attributes on the root directory"
```

Comment:
This warning is unreachable, just remove it.

Reply:
Fixed.